examples/itest/src/test/kotlin/stages/RuntimeServiceStages.kt

```diff
@@ -209,6 +215,15 @@ class RuntimeServiceActionStage : ActionStage<RuntimeServiceActionStage, Runtime
 
   }
 
+  fun execution_is_waiting_in_user_task(): RuntimeServiceActionStage = step {
+    while (taskService
```

Comment:
Shouldn't we go for awaitility instead and use `await().atMost().until {}` instead?
Thread sleep in JUnit Thread feels dirty to me.

Reply:
Now using awaitility